client/tests/unit/analyze-pending.spec.ts

```diff
@@ -1,16 +1,34 @@
 import { createLocalVue, mount } from '@vue/test-utils'
 import Vuetify from 'vuetify'
-import Comp from '@/components/new-request/analyze-pending.vue'
+import AnalyzePending from '@/components/new-request/analyze-pending.vue'
+import newReqModule from '@/store/new-request-module'
 
 const localVue = createLocalVue()
 const vuetify = new Vuetify()
 
 localVue.use(Vuetify)
 
+function setState () {
+  newReqModule.mutateEntityType('CR')
+  newReqModule.mutateLocation('BC')
+  newReqModule.mutateRequestAction('NEW')
+  newReqModule.mutateAnalyzePending(true)
+  return new Promise(resolve => {
```

Comment:
If you need an actual delay, I created a plugin, "sleep", for that purpose.

If you need to wait for Vue to finish its internal processing, `await Vue.nextTick()` is better.

But actually, here, you should just await each of the mutate calls (which are actually async). I tested this and it works :)

Reply:
good point!